packages/gatsby-plugin-treat/tests/styling.test.js

```diff
@@ -27,7 +27,7 @@ const gatsbyServerPort = 9999;
 const gatsbyBinaryPath = resolveBin('gatsby', 'gatsby');
 const gatsbyFixturePath = path.resolve(__dirname, '../../gatsby-plugin-treat-example');
 const gatsbyExecArgs = {
-  shell: 'bash', 
+  shell: '/bin/bash', 
```

Comment:
@mattsjones Was thinking we might be running into this

Reply:
Think it might be that!

https://travis-ci.org/seek-oss/treat/builds/625477966?utm_source=github_status&utm_medium=notification